index.js

```diff
@@ -1,3 +1,24 @@
+function showPopup() {
+  const node = $$("profile-button").getNode();
+  const isVisible = $$("mywindow").isVisible();
+  if (isVisible) {
+    $$("mywindow").hide();
+  } else {
+    $$("mywindow").show(node);
+
+  }
+}
+
+const profileList = {
+  view: "list",
+  width: 250,
+  height: 72,
```

Comment:
List has only 2 options, options have fixed height => better to replace `height` with `yCount: 2` or even `autoheight: true`
https://docs.webix.com/api__ui.list_ycount_config.html
https://docs.webix.com/api__ui.list_autoheight_config.html